### PR TITLE
display beta editor card

### PR DESCRIPTION
**Pull request by pelikhan**
- [x] display "Beta Editor" card in experiments, based on current URL (query and hash carried over)
- [x] deprecated "betaUrl" entry in pxtarget.json (ignored)

### CODE REVIEW

**Comment by abchatra** [comment]
I am not sure this will work on Minecraft. URL vastly differs between code connection, hour of code, in game and nether. Is it possible to just add /beta to the existing URL and #   and reload?

**Comment by abchatra** [comment]
Reload the existing URL with /beta inserted in between the URL.

**Comment by pelikhan** [comment]
Turns out that there were multiple places we were displaying this beta url string. I've deprecated that flag.

**Comment by pelikhan** [comment]
@abchatra

File: webapp/src/scriptsearch.tsx [hunk 1]
```diff
@@ -358,6 +357,11 @@ export class ScriptSearch extends data.Component<ISettingsProps, ScriptSearchSta
             && pxt.appTarget.appTheme.importExtensionFiles
             && !disableFileAccessinMaciOs
             && !searchFor;
+        // inject beta at end of / or /#
+        // also excludes http://localhost:port/index.html
+        const betaUrl = window.location.href.replace(/\/(#|$)/, "/beta$1")
```
Comment (abchatra) [inline webapp/src/scriptsearch.tsx]: Does this work with something like below? 
https://minecraft.makecode.com/?ingame=1&ipc=1&noRunOnX=1#tutorial:/hour-of-code/open-door

Comment (abchatra) [reply webapp/src/scriptsearch.tsx]: You should add for ? too?

**Pull request merged**